src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -567,6 +567,10 @@ public function mirror($originDir, $targetDir, \Traversable $iterator = null, $o
         }
 
         foreach ($iterator as $file) {
+            if ($file->getPathName() === $targetDir) {
```

Comment:
AFAICS, `$targetDir` is not normalized where `->getPathName()` always return a nomalized version of the path, so the comparison can fail for the same directory in many cases. Can we try to be more "robust"? Perhaps by using `realpath()`?